<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background: qconicalgradient(cx:0, cy:0, angle:137.7, stop:0 rgba(230, 255, 0, 86), stop:0.375 rgba(255, 255, 0, 69), stop:0.386364 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.505682 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.5625 rgba(255, 255, 0, 192), stop:0.619318 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>229</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Wilgotność    </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>230</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>201</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Jasność         </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>202</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>202</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>140</y>
      <width>31</width>
      <height>84</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>31</width>
      <height>84</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>173</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Temperatura </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>154</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string> Min         Max</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>174</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>174</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>31</width>
      <height>84</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Dodaj pokoj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b1">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>19</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Ustaw COM</string>
    </property>
   </widget>
   <widget class="QComboBox" name="c1">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>19</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 9pt &quot;Lato Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Grafika/grafika.qrc">:/new/prefix1/zar.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Grafika/grafika.qrc">:/new/prefix1/termometr.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Grafika/grafika.qrc">:/new/prefix1/krop.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Impact&quot;;</string>
    </property>
    <property name="text">
     <string>Sterowanie:</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>spinBox_3</zorder>
   <zorder>spinBox_4</zorder>
   <zorder>label_9</zorder>
   <zorder>label_8</zorder>
   <zorder>verticalSlider_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_10</zorder>
   <zorder>spinBox</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>label_7</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>pushButton</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>l1</zorder>
   <zorder>c1</zorder>
   <zorder>b1</zorder>
   <zorder>verticalSlider_3</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_6</zorder>
   <zorder>spinBox_5</zorder>
   <zorder>spinBox_6</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>label_11</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Grafika/grafika.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
